docs/api/rebac/groups.mdx

```diff
@@ -6,64 +6,111 @@ title: Groups API
+:::
 
-and the user will also get the role 'marketing#editor' in the resource 'marketing#social_media'.
+## Example Usage for User Groups:
 
+Let’s dive deeper into a simple use-case example for User Groups:
 
-## Create a Group via API
+Say our company has several teams, each with a few employees assigned to each team. Our goal is to allow all members of the Marketing Team to Edit Social Media resources based on their group membership.
 
-To create a new group, make a POST request to the following with the required data:
+For this purpose, we will first create a group named `teams`. The group, `teams` is a resource, while each specific team (Such as `marketing-team`) is a resource instance.
+
+Users are assigned to the `marketing-team` via the `member` role (The `member` role here is, of course, just an example. Feel free to customize roles as you wish).
+
+As mentioned above, we also have a `social-media` resource, and a `training_video` - which is a resource instance of the `social-media` type. This resource instance supports the role `editor`.
+
+Let’s set this up:
+
+### API Calls: Create Group, Add Role to Group, Add Users to Group:
+
+
+To **create a new group**, make a POST request with the following data:
 
 ```bash
 curl 'https://api.permit.io/v2/facts/{project_id}/{env_id}/groups' \
-  -H 'authorization: Bearer API_SECRET_KEY' \
-  --data-raw '{"group_instance_key": "Acme"}'
+ -H 'authorization: Bearer API_SECRET_KEY'
+ --data-raw '{"group_instance_key": "marketing", “group_resource_type_key": "teams"}' \
+```
+
+Create the `social-media` resource:
+
+```bash
+curl 'https://api.permit.io/v2/schema/{proj_id}/{env_id}/resources
+' \
+ -H 'authorization: Bearer API_SECRET_KEY'
+ --data-raw '{"key": "social-media", “name”: “social media”, “actions”:{}' \
 ```
 
-To add a new role to group, make a POST request to the following with the required data:
+Create a `training_video` resource instance:
+
+```bash
+curl 'https://api.permit.io/v2/schema/{proj_id}/{env_id}/resource_instances
+' \
+ -H 'authorization: Bearer API_SECRET_KEY'
+ --data-raw '{"key": "training_video", “tenant”: “business”, “resource”:social_media' \
+```
+
+Create a role called `editor`
+```bash
+curl 'https://api.permit.io/v2/schema/{proj_id}/{env_id}/roles' \
+ -H 'authorization: Bearer API_SECRET_KEY'
+ --data-raw '{"key": "editor", “name”: “editor” \
+```
+
+Next, we assign the `editor` role to the `marketing-team` group instance:
 
 ```bash
 curl 'https://api.permit.io/v2/facts/{project_id}/{env_id}/group/{group_resource_type_key}/roles' \
-  -H 'authorization: Bearer API_SECRET_KEY' \
-  --data-raw '{"resource": "marketing", "resource_instance": "social_media", "role": "editor", tenant:"business"}'
+ -H 'authorization: Bearer API_SECRET_KEY'
+ --data-raw '{"resource": "socail_media", "resource_instance": "training_video", "role": "editor", tenant:"business"}' \
```

Comment:
Like above.
The previous version was fine. 
and you have a mistake in social_media